Packages/com.unity.inputsystem/InputSystem/Devices/Precompiled/FastKeyboard.cs

```diff
@@ -24,12 +24,12 @@ internal partial class FastKeyboard : UnityEngine.InputSystem.Keyboard
         public const string metadata = ";AnyKey;Button;Axis;Key;DiscreteButton;Keyboard";
         public FastKeyboard()
         {
-            var builder = this.Setup(115, 15, 7)
+            var builder = this.Setup(127, 15, 7)
                 .WithName("Keyboard")
                 .WithDisplayName("Keyboard")
-                .WithChildren(0, 115)
+                .WithChildren(0, 127)
                 .WithLayout(new InternedString("Keyboard"))
-                .WithStateBlock(new InputStateBlock { format = new FourCC(1262836051), sizeInBits = 112 });
+                .WithStateBlock(new InputStateBlock { format = new FourCC(1262836051), sizeInBits = 136 });
```

Comment:
I use this is what we get for FourCC form the generator, just reacted to it but realised its generated :)